### PR TITLE
fix: using layout and semantic HTML

### PR DESCRIPTION
**Pull request by iamDistractech**
# Description

Closes 

Branch name:

## Type of change

Please select the option that is relevant

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

# Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] All the required documentation is added.
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.

# Additional information

### CODE REVIEW

File: src/lib/Env.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-// export const BASE_API_URL = import.meta.env.VITE_API_URL;
-export const BASE_API_URL = import.meta.env.VITE_STRAPI_API_URL;
+export const BASE_API_URL = import.meta.env.VITE_API_URL;
```
Comment (jochemvogel) [inline src/lib/Env.js]: Gaat dit nou werken met die Strapi api?

File: src/lib/GameList/GameListCard.svelte [hunk 2]
```diff
@@ -67,7 +67,7 @@
 	}
 
 	article h1 {
-		color: var(--color-dark-blue);
+		/* color: var(--color-dark-blue); */
```
Comment (jochemvogel) [inline src/lib/GameList/GameListCard.svelte]: ???

File: src/routes/__layout.svelte [hunk 3]
```diff
@@ -3,4 +3,40 @@
...
+			<li><a href="/spellen">Speloverzicht</a></li>
+			<li><a href="/">Inloggen</a></li>
+		</ul>
+	</nav>
+</header>
 <slot />
+
+<style>
+	header {
+		margin-top: 1.5rem;
```
Comment (jochemvogel) [inline src/routes/__layout.svelte:20]:
```suggestion
		margin-top: 1.5em;
```

Comment (iamDistractech) [reply src/routes/__layout.svelte:20]: nope, want ik wil niet dat de margin mee resized.

**Review by jochemvogel — APPROVED** [review]

**Comment by jochemvogel** [comment]
**MERGE DAN**

**Pull request merged**